httpsign/signer.go

```diff
@@ -79,7 +78,7 @@ type Signer struct {
 func New(config *Config) (*Signer, error) {
 	// config is required!
 	if config == nil {
-		return nil, fmt.Errorf("config is required.")
+		return nil, fmt.Errorf("config is required")
```

Comment:
Same as for https://github.com/mailgun/holster/pull/132#discussion_r984853331

Reply:
Reverted, added TODO.